crates/proof-of-sql-parser/src/utility.rs

```diff
@@ -98,6 +102,9 @@ pub fn div(left: Box<Expression>, right: Box<Expression>) -> Box<Expression> {
 /// Get table from schema and name.
 ///
 /// If the schema is `None`, the table is assumed to be in the default schema.
+/// # Panics
+///
+/// This function will panic if either the `name` or the `schema` (if provided) cannot be parsed.
```

Comment:
```suggestion
/// This function will panic if either the `name` or the `schema` (if provided) cannot be parsed as valid [Identifier]s.
```
The reason the "as valid Identifiers" is important is that it is not clear without that clause what would make it parsing not possible.

Please update this in other spots as well.